libgalois/include/katana/Properties.h

```diff
@@ -149,7 +149,7 @@ template <typename PropTuple>
 Result<std::tuple<>>
 ConstructPropertyViews(
     const std::vector<arrow::Array*>&, std::index_sequence<>) {
-  return std::tuple<>();
+  return Result<std::tuple<>>(std::tuple<>());
```

Comment:
`Result` constructor should handle this conversion. I'm surprised an explicit conversion is required. Did any build fail due to this? if so, maybe we should try to fix this in the `Result` constructor.

Reply:
The CUDA build fails if this header file is included into a `.cu` file compiled with `nvcc`. For some reason `nvcc` doesn't see the implicit constructor. We can't do much because `Result` definition simply redirects to Boost:
```
/// A Result is a T or an ErrorInfo.
template <class T>
using Result = BOOST_OUTCOME_V2_NAMESPACE::std_result<
    T, ErrorInfo, internal::abort_policy>;
```